hubble/src/postgres.rs

```diff
@@ -605,3 +605,80 @@ pub async fn insert_client_mapping<'a, A: Acquire<'a, Database = Postgres>>(
 
     Ok(())
 }
+
+pub async fn get_chain_ids_and_ids<'a, A: Acquire<'a, Database = Postgres>>(
+    db: A,
+) -> sqlx::Result<Vec<(String, i32)>> {
+    let mut conn = db.acquire().await?;
+
+    let rows = sqlx::query!("SELECT chain_id, id FROM v0.chains")
+        .fetch_all(&mut *conn)
+        .await?;
+
+    let chain_ids_and_ids: Vec<(String, i32)> =
+        rows.into_iter().map(|row| (row.chain_id, row.id)).collect();
+
+    Ok(chain_ids_and_ids)
+}
+
+pub async fn insert_or_update_tokens<'a, A: Acquire<'a, Database = Postgres>>(
+    db: A,
+    tokens: &[(i64, String, String, i64, Option<String>, String, bool)],
+) -> sqlx::Result<()> {
+    let mut conn = db.acquire().await?;
+
+    let (chain_ids, denoms, display_symbols, decimals, logo_uris, display_names, gas_tokens): (
```

Comment:
personally I wouldn't do the batch insert, but insert one-by-one: keeps come simpler and in this (low-volume) scenario good enough.

... but feel free to keep it as is